src/test/java/StringTest.java

```diff
@@ -0,0 +1,56 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
+
+class StringTest {
+    @Test
+    void splitTwoNumbersTest() {
+        String target = "1,2";
+        String delimiter = ",";
+        
+        String[] result = target.split(delimiter);
+        assertThat(result).contains("1", "2");
+    }
+
+    @Test
+    void splitOneNumberTest() {
+        String target = "1";
+        String delimiter = ",";
+
+        String[] result = target.split(delimiter);
+        assertThat(result).containsExactly("1");
+    }
+
+    @Test
+    void substringTest() {
+        String target = "(1,2)";
+
+        String result = target.substring(1, target.length() - 1);
+        assertThat(result).isEqualTo("1,2");
+    }
+
+    @Test
+    void charAtTest() {
+        String target = "abc";
+
+        char first = target.charAt(0);
+        assertThat(first).isEqualTo('a');
```

Comment:
```suggestion
        assertThat("abc".charAt(0)).isEqualTo('a');
```
위와 같이 구현하는 것은 어떨까?
너무 과할까?